data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/http/WebClientCloudEventSender.java

```diff
@@ -57,6 +63,9 @@ public final class WebClientCloudEventSender implements CloudEventSender {
   private final AtomicBoolean closed = new AtomicBoolean(false);
   private final AtomicInteger inFlightRequests = new AtomicInteger(0);
 
+  private final MeterRegistry meterRegistry;
+  private final Tags noResponseResourceTags;
```

Comment:
We should initialize these 2 fields in the constructor

Reply:
addressed